### ISSUE TITLE
what is the default location of camera?

### ISSUE THREAD

**Issue by liuhaoliang** [facebook/react-360#536]
Is the default location of camera is [0,0,0]?

**Comment by vineethariharan** [comment]
Yes. You can use the function r360.getCameraPosition() to fetch the camera position.
Or if that doesn't work(it didn't with me for some reason), you can access the ReactInstance class variable array r360._cameraPosition.
